pixelgl/window.go

```diff
@@ -119,6 +124,17 @@ func NewWindow(cfg WindowConfig) (*Window, error) {
 
 	w := &Window{bounds: cfg.Bounds, cursorVisible: true}
 
+	flag := false
```

Comment:
I thought about this solution but wouldn't this just check to see if its even?

Reply:
hahahaha yes it would.....brain fart. I think then, if there are specific values that can be set otherwise, it's an error, we should at least have those specific values documented.